pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -307,6 +307,11 @@
           "x-intellij-html-description": "arbitrary metadata ignored by <code>eksctl</code>.",
           "default": "{}"
         },
+        "eksctlCreated": {
```

Comment:
I think this shouldn't appear here but I see that `status` is already in here so it's ok, though we should hide that from the docs too